src/components/templates/NewsAndEvents.scss

```
@@ -0,0 +1,43 @@
+@import "../variables";
+
+.news-events__wrap {
+  background-color: $white_smoke;
+  margin: 0 auto;
+  padding-top: 56px;
+  padding-bottom: 56px;
+
+  @media screen and (min-width: $breakpoint-lg) {
+    display: flex;
+  }
```

```suggestion
  @media screen and (min-width: $breakpoint-lg) {
    display: flex;
    column-gap: 40px;
  }
```
こんな感じで、`column-gap`や`row-gap`を使うことで、子要素にpaddingを設定する必要がなく、すっきりします